ansible_collections/arista/avd/roles/eos_designs/doc/node-types.md

```diff
@@ -201,3 +201,21 @@ node_type_keys:
 
 !!! info
     The default node definition is available in the [default section](../defaults/main/main.yml) of the eos_designs role.
+
+## Default Node Types
+
+Node types can be defined statically on each node or in each group of nodes.  As an alternative to this, regular expressions can be used to determine the node type based
+on the hostname.
+
+Please note that using the `default_node_types` functionality will cause certain tests in the eos_validate_state role to not be executed.  This functionality will be restored as part of a later update to eos_validate_state and this note will then be removed.
```

Comment:
I would add some warning annotation here r something to stand out on the page - cf Warning there: https://avd.sh/en/devel/roles/eos_designs/doc/fabric-variables.html

Reply:
Good point - thanks for that.